### PR TITLE
testing unit test - in pipeline

### PR DESCRIPTION
**Pull request by PratikShelke-1**

### CODE REVIEW

File: README.md
Comment (L00163597) [inline README.md:1]: This is fine to keep on main

File: Source/grassroots/__pycache__/tests.cpython-310.pyc
Comment (L00163597) [inline Source/grassroots/__pycache__/tests.cpython-310.pyc:1]: Same again, merge or remove

File: Source/grassroots/migrations/0001_initial.py
Comment (L00163597) [inline Source/grassroots/migrations/0001_initial.py:1]: Unsure if this is needed

File: Source/grassroots/migrations/__pycache__/0001_initial.cpython-310.pyc
Comment (L00163597) [inline Source/grassroots/migrations/__pycache__/0001_initial.cpython-310.pyc:1]: Same as above

**Review by L00163597 — CHANGES_REQUESTED** [review]
Make these below changes:

- [x] Move ed2deploy file to it's own branch until secrets are added
- [ ] Merge this branch from Develop. There is a lot of files here that are already in Develop branch, making it look like you are adding new files, which you are not.

Overall, unsure if the migration files are needed. Merge from Develop is needed to keep this branch up to date. I changed this branch's base to Develop, as main should not be directly merged into from a feature branch.

File: Source/grassroots/test_django.py
[the remark this answers is not in the document]
Comment (PratikShelke-1) [reply Source/grassroots/test_django.py:1]: This is done. I have put it in the test folder and also i have deleted the file from main

File: Source/grassroots/tests.py
[the remark this answers is not in the document]
Comment (PratikShelke-1) [reply Source/grassroots/tests.py:1]: Done

**Comment by PratikShelke-1** [comment]
> Make these below changes:
> 
> * [x]  Move ed2deploy file to it's own branch until secrets are added
> * [x]  Merge this branch from Develop. There is a lot of files here that are already in Develop branch, making it look like you are adding new files, which you are not.
> 
> Overall, unsure if the migration files are needed. Merge from Develop is needed to keep this branch up to date. I changed this branch's base to Develop, as main should not be directly merged into from a feature branch.

Above changes has been done as requested

File: .github/workflows/ec2deploy.yml
Comment (L00163597) [inline .github/workflows/ec2deploy.yml:1]: This is only working on your fork right now, so put this in it's own branch. If you already have this in it's own branch, delete it from here

File: Source/grassroots/__pycache__/test_django.cpython-310.pyc
Comment (L00163597) [inline Source/grassroots/__pycache__/test_django.cpython-310.pyc:1]: Remove all __pycache__ folders and their contents

Comment (L00163597) [inline Source/grassroots/__pycache__/test_django.cpython-310.pyc:1]: Remove all pycache folders and their contents

**Review by L00163597 — CHANGES_REQUESTED** [review]
Make sure all pycache folders are deleted with their content from all folders

**Comment by PratikShelke-1** [comment]
Done. Please check now

**Pull request merged**